libraries/Microsoft.Bot.Builder.Ai.QnA/Microsoft.Bot.Builder.Ai.QnA.csproj

```diff
@@ -39,16 +39,28 @@
 
 	<PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Documentation|AnyCPU'">
 		<DocumentationFile>bin\$(Configuration)\netstandard2.0\Microsoft.Bot.Builder.Ai.QnA.xml</DocumentationFile>
+		<CodeAnalysisRuleSet>Microsoft.Bot.Builder.Ai.QnA.ruleset</CodeAnalysisRuleSet>
 	</PropertyGroup>
 
 	<PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
 		<DocumentationFile>bin\$(Configuration)\netstandard2.0\Microsoft.Bot.Builder.Ai.QnA.xml</DocumentationFile>
+		<CodeAnalysisRuleSet>Microsoft.Bot.Builder.Ai.QnA.ruleset</CodeAnalysisRuleSet>
+	</PropertyGroup>
+
+	<PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+	  <CodeAnalysisRuleSet>Microsoft.Bot.Builder.Ai.QnA.ruleset</CodeAnalysisRuleSet>
+	</PropertyGroup>
+
+	<PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug - NuGet Packages|AnyCPU'">
+	  <CodeAnalysisRuleSet>Microsoft.Bot.Builder.Ai.QnA.ruleset</CodeAnalysisRuleSet>
 	</PropertyGroup>
 
 	<ItemGroup>
+    <PackageReference Include="AsyncUsageAnalyzers" Version="1.0.0-alpha003" />
     <PackageReference Include="Microsoft.Bot.Builder" Condition=" '$(PackageVersion)' == '' " Version="4.0.0-local" />
```

Comment:
Same comments on this .csproj file as on the Azure file.

Reply:
Thanks @cleemullins. I applied your feedback.